rpsl-parser/irr.go

```diff
@@ -257,7 +258,7 @@ func (r *Reader) readValue() (string, bool, error) {
 
 		switch {
 		// newline and letter, return value.
-		case IsNewline(ch) && IsLetter(r.Peek()):
+		case IsNewline(ch) && unicode.IsLetter(r.Peek()):
```

Comment:
Is this 'IsLetter()' just going to do ascii letters?
(is RPSL at all non-ascii? non-7-bit I mean)

Reply:
https://play.golang.org/p/hrB5BSWXmaE

seems to say that 'letter' is really a-zA-Z and not numbers, cool.